src/app/mv-wrapper/media-viewer-wrapper.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, Compiler, ViewChild, ViewContainerRef, ComponentFactoryResolver,
+  ComponentFactory, ElementRef, OnInit, Renderer2, Inject, PLATFORM_ID } from '@angular/core';
+import { isPlatformBrowser } from '@angular/common';
+import { WindowService } from '@hmcts/ccd-case-ui-toolkit/dist/shared/services/window';
+
+const MEDIA_VIEWER = 'media-viewer';
+
+@Component({
+  selector: 'ccd-media-viewer',
+  templateUrl: './media-viewer-wrapper.component.html'
+})
+export class MediaViewerWrapperComponent implements OnInit {
+
+  mediaURL = '';
+  mediaFilename = '';
+  mediaContentType = '';
+
+  public constructor(public renderer: Renderer2,
+                     private el: ElementRef,
+                     private windowService: WindowService,
+                     private readonly viewContainerRef: ViewContainerRef,
+                     private componentFactoryResolver: ComponentFactoryResolver,
+                     @Inject(PLATFORM_ID) private platformId: Object) {
+  }
+
+  async ngOnInit() {
+
+    console.log('Platform ID is ' + this.platformId);
+
+    if (!isPlatformBrowser(this.platformId)) {
+      console.log('On server. Skipping.');
+      return;
+    }
+
+    const localStorageMedia = this.windowService.getLocalStorage(MEDIA_VIEWER);
+    if (localStorageMedia) {
+      const media = JSON.parse(localStorageMedia);
+      this.mediaURL = media.document_binary_url;
+      this.mediaFilename = media.document_filename;
+      this.mediaContentType = media.content_type;
+    }
+    // this.windowService.removeLocalStorage(MEDIA_VIEWER);
+    this.removeSideBar();
+  }
+
+  removeSideBar() {
+    if (this.el.nativeElement) {
+      let tempElement = this.el.nativeElement.parentElement
+      while (tempElement) {
+        if (tempElement.tagName === 'HTML') {
+          this.renderer.setStyle(tempElement, 'background-color', '#ffffff');
+        }
+        tempElement = tempElement.parentElement;
+      }
+    }
+  }
+}
```

Comment:
The ` MediaViewerWrapperComponent ` is moved from case-ui-toolkit to case-man-web to apply the SSR exclusion in order to fix the build fail in pipeline.    This may be a risk if in future, this component is moved back to toolkit again. 
@mario-paniccia @andrewfolga Inline to ExUI requirement, will this be a risk?

Reply:
ExUI has similar dependency versions, due to which, most probably the same tricks will be applied in ExUI code.